README.md

```
@@ -462,6 +462,45 @@ This heading will use a custom component!`;
 />
 ```
 
+## 📦 Bundle Optimization
+
+Streamdown is optimized for minimal bundle size by making heavy components **opt-in**. By default, Code blocks, Mermaid diagrams, and Math expressions render as lightweight fallbacks (plain text). To enable full functionality, import and pass the components you need:
+
+### Enabling Heavy Components
+
+```svelte
+<script>
+	import { Streamdown } from 'svelte-streamdown';
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)